### DISCUSSION TITLE
New PvP mode

### DISCUSSION THREAD

**Discussion by 55Honey** [chromiecraft/chromiecraft#3750] [PTR Feedback]
Tonight we've tested a new PvP mode on the PTR. Please provide your feedback about this event. Every detail can help, thanks!

For reference, these spells were disabled:
Healing Touch, Rejuvenation, Regrowth, Swiftmend, Flash of Light, Holy Light, Lay on Hands, Holy Shock, Flash Heal, Greater Heal, Heal, Lesser Heal, Renew, Penance, Chain Heal, Earth Shield, Healing Wave, Lesser Healing Wave, Riptide, Revenge, Hammer of Righteous.

Currently Players receive 1 score per creature, 20 per PvP kill and lose 10 per PvP death.

**Comment by Hihilist** [comment]
So far experienced issues:

1. You may keep trinkets after HSing. You may keep potions after mode is done.
3. Your HP may not be adjusted to the mode so you will join with normal HP.
4. HSing outside mode may not clear phasing and you may end up in 'phase shuffle' in city = no NPCs.
5. Mode timer at the top of the screen may vanish and does not reappear after rejoining.

**Comment by plasma-93** [comment]
Better instructions next time would help a lot!

Random feedback from the quick test:

I read the gossip from the NPC and he was bugged somehow to not show me the proper dialogue with teleports to Loch Modan until I relogged.

If I read correctly, that Hammer of the Righteousness and Revenge are disabled? If so just delete those classes from the mode kekw

There were a lot of people I just couldn't attack for some reason, Barcode and Alexsander (guildies) were the 2 I could never attack when I saw them

Whoever gets the kill should get the points, stealth leeching was weirdge

**Comment by Gultask** [comment]
- Ending the event properly removed the trinket and neck pieces, but I've seen people say that they managed to keep theirs?
- FFA flag seems to have some problems, since it's not being applied 100% of the time. But the second try we had worked perfectly.
- I can buff myself with potions and scrolls, I don't know if that's intended (I don't PvP). I have to rebuff myself if I die or zone in, though.
- You can keep the potions after leaving.
- Timer also disappeared for me. I think HS'ing might've caused that.

Suggestion:

- Maybe add a some kind of aura like Hunter's Mark to the top player, or to anyone with high rank. Since players lose points when they die it could add more tension to the match if they're singled out.

Edit: spell/aura 46459 is a 2 min debuff that to my knowledge does nothing but add a marker.

**Comment by 55Honey** [comment]
Thanks everyone for the feedback here and elsewhere.

Unfortunately players managed to bug out some part of the script at the start due to the use of GM commands. While we pushed some code fixes to prevent this from happening, this is an oversight on our end and will not be possible with future tests.

Based on the input we got, the following changes were committed:
- all remaining healing effects are reduced by 50%
- fixed a bug allowing players to keep temporary items
- the score calculation for creep kills is now only based on who delivered the killing blow
- the range to share score for PvP kills was reduced from 50m to 35m and only visible players are able to receive score

We will keep working on it and testing it and get back to you when we're ready for another round. Once again, thanks for your time.

**Comment by 55Honey** [comment]
Further work has been done on this subject. I plan to announce an appointment for another test soon.

- players can now leave the tournament with their Hearthstone
- if a player re-joins the same tournament after they had used their Hearthstone their score and previously earned items will be restored
- players who leave the allowed area will receive a warning every second for 10 seconds. If they fail to return to the tournament area, they will be randomly teleported and have their hitpoints set to 1.
- the list of forbidden spells was updated:
Healing Touch, Rejuvenation, Regrowth, Swiftmend, Flash of Light, Holy Light, Lay on Hands, Holy Shock, Hammer of Righteous, Flash Heal, Greater Heal, Heal, Lesser Heal, Renew, Penance, Chain Heal, Earth Shield, Healing Wave, Lesser Healing Wave, Riptide, ~~Revenge~~, new: Shield Bash, new: Shield Slam.
- Due to the concerns raised that durable classes and specs which are good at AoE fights would just pull the whole map, all creeps have received a soft enrage after 30 seconds.

As always, thanks for catching up on this subject. Any and all feedback is much appreciated.

**Comment by Pinio1** [comment]
First of all i appreciate this idea and it seems that this could be nice event.

Although tuning is a joke. It seems you are not aware of current pvp balance issues.

Tuning as you described means that this event will be hunter/warrior/rogue/retri fiesta. All casters you can literally ban from participating.     

How you can make x3 HP but leave mana without changes? Imagine lock life tapping all the time cause he wont have enough mana to kill x3 HP warrior - same as boomkin, ele shammy and Shadow priest.  Mages frost shield will be useless as well.                                                    But yes we have retri, hunters who will be much more overpowered than they are now.

What  is x3 HP for ? Since we have R14 rewards  basing on HK you should encourage ppl to join this event to have possibility to obtain large amount of HK.  However, x3 HP is opposite to that. PVP event should help to obtain PVP rewards...

Some spec like ele shammy/boomkins/destro lock base on burst damage meaning that they can 2-3 shot ppl if they play it correcrtly. They do not stand any chance in this event against other classes  (hunter/warrior/rogue/retri)  as they wont stand long 1 vs 1 fight with double x3 hp since their nuke is gone in this event.

**Comment by Fibri123** [comment]
First of all i want to make it clear that the fact some dev took their free time to make an custom event for players is awesome, full respect and admiration. 

WoW was never a very well balanced game at any point during any expansion, and world pvp is even wrose in that sense, the adjustment made here dont help at all and make it even worse. still some people can enjoy it that's cool. however the rewards for this will essentialy force plenty of PvE players to suffer through this and they will be frustrated(since there is no other alternative). giving exclusive PvE mogs for PvP activity on a RP-PvE sever feels like blizzard annoucing diablo immortal, Extremly out of touch.

**Comment by Never6** [comment]
I think players should have their damaged increased obviously do not increase by x3 but somewhere in the middle maybe 2x maybe 1.5. Who knows that will come down to testing.
But what i do know is that if there is no damage increase, Burst classes get hard screwed over by health increases since those classes cannot do 7k+ in a global. And if you ask me increasing players health is only gonna want them to play a tank more. Like druid for example why would i bother playing cat in this when i can just play bear (its basically the same talents anyway) but i have way more HP Armor And bear still does do decent damage ive seen 1.4k mauls possible even without rend and tear
Prot warriors with x3 health will have 18k self buffed in bis gear right now. They will still have their damage from revenge procs just no purge +1k free ability with shield slam and they will have to switch to berzerker stance to kick OH THE HORROR!!!!! 
I can kill him But the time it takes for me to kill him someone is gonna come help because its world pvp.
Also i didnt see this anywhere in the post but maybe i just missed it. I think players should get tokens for killing other players. Or have special transmogs that can only be bought with tokens gained from killing players something along these lines.
I think players should be rewarded for trying to find people and also winning 1v1s or 1vXs

**Comment by leastretardedccplayer** [comment]
Hey, so I just recently tested the new PVP mode and overall liked the idea of the game mode, but some small QoL changes would make it better imo:
- First, when you respawn, you can be instantly attacked by 3-5 mobs in your respawn zone, so I personally think a small addition of a short invulnerability/invisibility buff would be great to give you time to react to your surroundings.
- Second, I found that killing the mobs feels completely irrelevant, although I like their presence, they don't drop anything of value to the game mode itself, other than normal food. I feel like adding more advanced food (the ones that gives temporary buffs), elixirs, or weapon oils/sharpeners to the drop pool would be good addition for the overall "battleroyale" PvP aesthetic and make killing mobs more rewarding at least early on.
- I'd also like to suggest that when you deal a killing blow to an enemy, all negative DoTs should be removed from you, and maybe you should get some kind of little healing in the form of a HoT or something. It might be a big change, but I'd like to suggest it nonetheless.
- Another point I have is about the spawn points themselves. Some are extremely unpleasant to appear in, such as mines, caves, etc. You can just randomly spawn in one of them (sometimes even at the very end of them), and the amount of mobs there is simply unbearable, so you would have to spend a good 5 minutes just getting out of there. I personally think that such spawn points should re-reviewed. They can be removed from the spawn pool as the chances of any pvping giong there is very unlikely (for the reason mentioned earlier) or you can consider lowering the amount of NPC mobs in those locations, but I still prefer the first solution.
- And finally, as much as I hate to suggest it (being a rogue player myself), I think some stealth detection orbs/buffs (similar to the ones in the arena) should be implemented, because it's easy for me to see that this mode really benefits camping classes like druid/rogue as you can just sneak through everything and just pick off anyone who's at low on health. Now, orb balance is a very touchy subject indeed, maybe the detection range should be reduced along with the duration, and maybe the orbs themselves should be rare to find (maybe they can drop from mobs, which would be a good addition to my second point).

That's all I can think of for now. Again, I'd like to say that overall I enjoyed the experience, even if it was short one. Thank you for creating new content on the server.

EDIT1: Forgot to mention one more thing. Maybe all players should start in some kind of same starting gear. I'm currently farming HKs for my r14 gear and in this mode you can clearly see the difference in HP for those with BWL/R14 gear and those without, and with 3x HP it makes a big difference. So maybe to even the odds, all players should start with the same set of temporary gear.

**Reply by 55Honey** [reply]
Thanks for the honest and productive feedback. Your points will be considered in the design process!

**Reply by 55Honey** [reply]
>  I found that killing the mobs feels completely irrelevant, although I like their presence, they don't drop anything of value to the game mode itself

Please note that killing these creatures has a 10% chance to grant you a token to turn in for transmogs appearances at Honey Boatwrighte in Stormwind or Orgrimmar. It is the only source of obtaining these tokens.

I think replacing the full gear of a player is beyond our scope - at least for now.

**Reply by leastretardedccplayer** [reply]
Yes, I'm aware of the tokens that drop after killing any NPC mobs and I think that's a good reason to farm those, but my point was that they don't drop anything that helps you during the actual tournament, so there's no reason to kill those mobs (unless you're obviously keen on farming tokens). I guess my question in that case would be, "What would be the point of killing these mobs if I collected enough tokens for every transmog?"

**Reply by 55Honey** [reply]
They still give you 1 score per kill then. Which is relevant to determine the winner(s).

**Reply by leastretardedccplayer** [reply]
As for changing player gear - I fully understand that there could be a limitations to that. I just wanted to add that point so you are at least aware of it. Thanks for the replies anyway.

**Comment by Hihilist** [comment]
Not sure if you want feedback/suggestions for the mode here or in the separate thread in Discussions but anyway, here are a couple of additions that I would like to present, so they can be considered:
- allow to play the mode more than one day in a week (but still only twice per day)
total crazy option - for the entire duration of Call to Arms
reasonable option - add Sunday to pair with Saturday

_Reasoning - limiting the availability of the mode to just one day does not help with making the mode popular and having just only one day to experience it ignores the fact that people may not be free to play on Saturday and/or in the specific timeslots_

- allow Honey NPC to exchange the tokens and explain the rules the whole week, not just the Saturday

_Reasoning - without these two options new players may not be able to learn that mode even exists, what are the rules and what are the rewards_

- change NPC appearance to something more outstanding

_Reasoning - right now NPC with current look and just gossip blends too much with regular NPCs and does not catch interest of players to actually try the mode_

- add shields and off-hands model to the possible rewards for tokens

_Reasoning - offering wands but not very noticeable shields or off-hands does not look very appealing and kind of ignore tanks, shield-bearers and casters, while wands are the least shown weapons right now_

**Reply by 55Honey** [reply]
Hi @Hihilist, thanks for the feedback.
Currently the brawl is deactivated due to a crash we're dealing with and could not ultimately identify now. I am the only one directly working on it, so no dev-time is used for custom content. I do so only in my free time,  which is rare currently, so it might take a while to get in back to a playable state.
For that reason the Honey NPC was respawned for the time being. It is meant to be available all week long.

I'm sorry for the inconvenience.

I'll take notes about the other points you've mentioned and they will be considered in the development process.

Cheers!
